packages/jest-create-cache-key-function/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../tsconfig.json",
```

Comment:
this should have a `references` field for the types package. Something like https://github.com/facebook/jest/blob/abf9f8d5e2a9032fec200174db9934f2df53b2fd/packages/jest-changed-files/tsconfig.json#L7-L9